app/publishing/handle-publish-reasoning.js

```diff
@@ -8,7 +8,7 @@ const handlePublishReasoning = (error) => {
       return INVALID
     default:
       console.log(`Publish fail reason ${error.message} not recognised`)
-      return undefined
+      return UNSUCCESFUL
```

Comment:
adding this because returning undefined stops the error being written to the database